CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+# 🙌 Contribution Guidelines
+
+**First of all thanks for taking time and interest in the project 👍**
+
+## First time Contributors
+If you're new to open source and don't know how to contribute to open-source projects then check out this [handy-guide](https://github.com/firstcontributions/first-contributions) 
+
+**If you want to contribute start with [git](https://git-scm.com/) by installing it on your machine**
+
+## Forking the Repository
+- Start by clicking the fork button in this repository
+
+## Copy and Clone
+- Copy the url of the repository
+- Clone the repository using the command in terminal 
+```
+git clone 'url you just copied'
+```
+## Commit
+- Commit the changes to the new branch using command 
+```
+git commit -m 'message'
+```
+
+## Push changes
+- Push changes on your local machine to GitHub using command 
+```
+git push origin master
+```
+
+## Compare and Pull Request
+- Compare the branches and start a Pull Request to merge the changes in the master branch
+
+## You contributed to the project. Congrats on your contribution 🎉 and welcome to Open Source 
```

Comment:
Line too long for a section heading

Reply:
Fixed section heading check now @Vibhu-Agarwal